src/main.rs

```diff
@@ -63,7 +73,17 @@ fn main() -> Result<()> {
                 }
             };
             let env = matches.value_of("env").map(|v| v.to_owned());
-            println!("{}", reqq.execute(req, env)?);
+            let req_args = matches.values_of("arg").unwrap();
```

Comment:
If we have no extra args passed, this crashes. We'll need to see if any were passed first.

Reply:
Hmm, that might be helpful to handle lol. I'll get that fixed here in a little bit.